### PR TITLE
feat: admin manage user

### PR DESCRIPTION
**Pull request by namdnhn**

### CODE REVIEW

**Review by KhoaAnhDoNguyen — APPROVED** [review]

**Review by letanhoaUIT — APPROVED** [review]

**Comment by namdnhn** [comment]
ready

File: BackEnd/app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -71,8 +71,10 @@ public function update(Request $request, $id)
     public function destroy($id)
     {
         $user = User::find($id);
+        logger("oke");
```
Comment (ghost) [inline BackEnd/app/Http/Controllers/UserController.php]: xóa log này

File: BackEnd/app/Http/Controllers/UserController.php [hunk 2]
```diff
@@ -71,8 +71,10 @@ public function update(Request $request, $id)
...
         if ($user) {
-            $user->delete();
+            $user->update(['is_active' => false]);
+            logger("oke 2");
```
Comment (ghost) [inline BackEnd/app/Http/Controllers/UserController.php]: log

**Review by KhoaAnhDoNguyen — APPROVED** [review]

**Comment by namdnhn** [comment]
ready

**Review by letanhoaUIT — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**